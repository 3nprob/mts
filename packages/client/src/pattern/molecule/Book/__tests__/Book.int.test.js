import { defaultRender } from './Book.int.render'

describe('<Book />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
