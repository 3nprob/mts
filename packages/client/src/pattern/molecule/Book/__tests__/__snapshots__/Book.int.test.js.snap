// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Book /> Snaphot must match defaultRender 1`] = `
<div
  className="topBorder"
>
  <div
    className="container1"
  >
    <div
      className="container1Image"
    >
      <div
        style={
          Object {
            "paddingBottom": "75%",
            "width": "100%",
          }
        }
      />
      <picture>
        <img
          alt="banner"
          src={Object {}}
        />
      </picture>
    </div>
    <div
      className="container1Info"
    >
      <h2>
        Will Smith - A Biography
      </h2>
      <p
        className="container1InfoLead"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien.
      </p>
      <div
        style={
          Object {
            "marginBottom": "10px",
          }
        }
      />
      <div>
        <a
          className="container1InfoLink"
          href="https://www.amazon.com/Will-Smith-Biography-Greenwood-Biographies/dp/0313376107"
          rel="noopener noreferrer"
          target="_blank"
        >
          Get the BookÂ 
          <svg
            height="10px"
            viewBox="0 0 185.343 185.343"
            width="10px"
          >
            <path
              d="M51.707 185.343a10.692 10.692 0 0 1-7.593-3.149 10.724 10.724 0 0 1 0-15.175l74.352-74.347L44.114 18.32c-4.194-4.194-4.194-10.987 0-15.175 4.194-4.194 10.987-4.194 15.18 0l81.934 81.934c4.194 4.194 4.194 10.987 0 15.175l-81.934 81.939a10.678 10.678 0 0 1-7.587 3.15z"
              fill="#010002"
            />
          </svg>
        </a>
      </div>
      <div
        style={
          Object {
            "marginBottom": "10px",
          }
        }
      />
      <small>
        <a
          href="https://www.amazon.com/Will-Smith-Biography-Greenwood-Biographies/dp/0313376107"
          rel="noopener noreferrer"
          target="_blank"
        >
          Get it on Amazon.
        </a>
      </small>
    </div>
  </div>
</div>
`;
